<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE TS><TS version="2.0">
<context>
    <name>CouldntLoadPluginMessage</name>
    <message>
        <location filename="ScriptExtenderPluginChecker.py" line="116"/>
        <source>Couldn&apos;t load {0} ({2}). The last error code was {1}.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="ScriptExtenderPluginChecker.py" line="114"/>
        <source>Couldn&apos;t load {0} ({2}). A DLL is invalid (code {1}).</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="ScriptExtenderPluginChecker.py" line="112"/>
        <source>Couldn&apos;t load {0} ({2}). A dependency DLL could not be found (code {1}). {3}</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>NormalPluginMessage</name>
    <message>
        <location filename="ScriptExtenderPluginChecker.py" line="71"/>
        <source>loaded correctly</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="ScriptExtenderPluginChecker.py" line="81"/>
        <source>reported as incompatible during query</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="ScriptExtenderPluginChecker.py" line="82"/>
        <source>reported as incompatible during load</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="ScriptExtenderPluginChecker.py" line="74"/>
        <source>disabled, fatal error occurred while loading plugin</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="ScriptExtenderPluginChecker.py" line="76"/>
        <source>disabled, no name specified</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="ScriptExtenderPluginChecker.py" line="83"/>
        <source>disabled, fatal error occurred while checking plugin compatibility</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="ScriptExtenderPluginChecker.py" line="84"/>
        <source>disabled, fatal error occurred while querying plugin</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="ScriptExtenderPluginChecker.py" line="66"/>
        <source>{0} version {1} ({2}, {4}) {3}.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="ScriptExtenderPluginChecker.py" line="78"/>
        <source>disabled, incompatible with current runtime version</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="ScriptExtenderPluginChecker.py" line="79"/>
        <source>disabled, requires newer script extender</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>NotAPluginMessage</name>
    <message>
        <location filename="ScriptExtenderPluginChecker.py" line="134"/>
        <source>{0} ({1}) does not appear to be a script extender plugin.</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>PluginMessage</name>
    <message>
        <location filename="ScriptExtenderPluginChecker.py" line="31"/>
        <source>{0} ({1}) existed.</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>ScriptExtenderPluginChecker</name>
    <message>
        <location filename="ScriptExtenderPluginChecker.py" line="182"/>
        <source>Checks script extender log to see if any plugins failed to load.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="ScriptExtenderPluginChecker.py" line="202"/>
        <source>Script extender log reports incompatible plugins.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="ScriptExtenderPluginChecker.py" line="207"/>
        <source>You have one or more script extender plugins which failed to load!

 If you want this notification to go away, here are some steps you can take:
  &#xe2;&#x80;&#xa2; Look for updates to the mod or the specific plugin included in the mod.
  &#xe2;&#x80;&#xa2; Disable the mod containing the plugin.
  &#xe2;&#x80;&#xa2; Hide or delete the plugin from the mod.

To refresh the script extender logs, you will need to run the game and/or editor again!

The failed plugins are:{0}</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="ScriptExtenderPluginChecker.py" line="176"/>
        <source>Script Extender Plugin Load Checker</source>
        <translation type="unfinished"></translation>
    </message>
</context>
</TS>
